### ISSUE TITLE
Drone Chassis

### ISSUE THREAD

**Issue by tomgco** [droneracingleague/technical-regulations#2]
Everyone starts with the same drone chassis? Or perhaps this could be relaxed to be within certain dimensions?

**Comment by rudids** [comment]
I think we have a standard chassis, but we allow people to decide on composites?  What are your views?

**Comment by sublimino** [comment]
Agreed - set dimensions/chassis but composite innovation permitted. Also
could be hard to police materials.

**Comment by tomgco** [comment]
> I think we have a standard chassis, but we allow people to decide on composites? What are your views?

I like the idea of composites and a standard chassis that falls within the dimensions specified.
